src/components/HotKey/HotKey.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {ReactNode} from 'react';
+import {block} from '../utils/cn';
+import type {DOMProps, QAProps} from '../types';
+
+import {Platform} from './types';
+import {defsByPlatform} from './definitions';
+import {parseKeyGroups} from './parse';
+import {isMac} from './utils';
+
+import './HotKey.scss';
+
+const b = block('hotkey');
+
+const Spaces = {
+    NoBreakSpace: String.fromCharCode(160), // &nbsp;
+    WordJoiner: String.fromCharCode(8288), // &NoBreak;
+};
+
+export interface HotKeyProps extends DOMProps, QAProps {
+    /**
+     * @example
+     * 'mod+a mod+c mod+v'
+     */
+    value: string;
+    platform?: Platform;
+}
+
+export const HotKey = React.forwardRef<HTMLElement, HotKeyProps>(function HotKey(props, ref) {
```

Comment:
> on mac Shift in the middle of expression for some reason

It's correct. In macOS Shift must be before Command

U can see priorities by platform in `definitions.ts`

Reply:
> Please add tests for pc/mac parse results display, example in story doesn't looks correct

I added parsing tests without linking to a specific platform

Presumably, we shouldn't want to have platform-specific tests, because they are essentially config tests.

Reply:
> In macOS Shift must be before Command

where i can read about this?

> we shouldn't want to have platform-specific tests

i mean to pass different `platform` in component, not running tests in VM with macOS